tests/e2e/conftest.py

```diff
@@ -383,6 +383,7 @@ def run_cli(
         arguments: List[str],
         *,
         verbosity: int = 0,
+        raise_for_returncode: bool = True,
```

Comment:
Please don't add the argument.
Catching `subprocess.CalledProcessError` can be used instead.

Reply:
my motivation to change the `helper` was that when one catches `subprocess.CalledProcessError` via `pytest.raises`, there's no way that I know to assert stderr (`pytest.raises(..., match="text")` will match `text` only to stdout, however the HTTP trace is printed to stderr).

Reply:
however, if we drop e2e tests as it's proposed in another comment, we don't need this functionality at all.

Reply:
though I'm still curious how to assert on stderr with pytest if an exception was thrown.

Reply:
RTFD, dude: https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError

Reply:
So, please revert the file entirely.

Reply:
ok

Reply:
sorry forgot to revert the helpers as well.